src/main/java/team/reborn/energy/api/base/CreativeEnergyStorage.java

```diff
@@ -0,0 +1,38 @@
+package team.reborn.energy.api.base;
+
+import net.fabricmc.fabric.api.transfer.v1.transaction.TransactionContext;
+import team.reborn.energy.api.EnergyStorage;
+
+/**
+ * An energy storage that can't accept energy, but will allow extracting any amount of energy.
+ * Creative batteries are a possible use case.
+ * {@link #INSTANCE} can be used instead of creating a new object every time.
+ */
+public class CreativeEnergyStorage implements EnergyStorage {
```

Comment:
Would InfiniteEnergyStorage be a better name?

Reply:
Yes why not, updated.